sites/partners/src/applications/PaperApplicationForm/sections/FormPreferences.tsx

```diff
@@ -43,21 +45,71 @@ const FormPreferences = ({ county, preferences, hhMembersOptions }: FormPreferen
 
   const watchPreferences = watch(allOptionFieldNames)
 
-  function uncheckPreference(metaKey: string, options: FormMetadataOptions[]) {
-    const preferenceKeys = options?.map((option) => option.key)
-    preferenceKeys.forEach((k) => setValue(getPreferenceOptionName(metaKey, k), false))
-  }
+  const exclusiveKeys = getExclusiveKeys(preferences)
 
   if (!hasMetaData) {
     return null
   }
 
+  const getExclusiveOption = (label: string, noneKey: string, preference: Preference) => {
+    return (
+      <Field
+        id={noneKey}
+        name={noneKey}
+        type="checkbox"
+        label={label}
+        register={register}
+        inputProps={{
+          onChange: (e) => {
+            if (e.target.checked) {
+              setExclusive(true, setValue, exclusiveKeys, noneKey, preference)
+            }
+          },
+        }}
+      />
+    )
+  }
```

Comment:
I feel like I'm missing something, but does it make sense to start saving the value of the "none" checkboxes? In the case of Coliseum, and maybe future cases, wouldn't it be useful to know the distinction? Then the only real difference between these "none" types and the rest, would be with setExclusive, and a lot of this could be simplified.

Same goes for the public site's handling of this.

Reply:
The distinctions between a normal and exclusive option if you look at the public implementation is the onChange, the validation, and the optional existence of both a description and a household member dropdown. I feel like it's cleaner to separate them out into two fxns than have one with an "if exclusive" in four places, but I don't feel strongly about this. And the values still get sent through the form, so I'm not sure what you mean by starting to save the values?

Reply:
When I check "None of these preferences apply to me, but I would like to be considered" or "
I don't want to be considered for [housing authority] project-based voucher units" those values are not saved in the application.

Reply:
Taking offline (online?)